src/components/folders/FolderItem.jsx

```diff
@@ -1,125 +1,33 @@
-import {
-  chakra,
-  LinkOverlay,
-  LinkBox,
-  Divider,
-  HStack,
-  Icon,
-  Text,
-  Box,
-} from "@chakra-ui/react"
-import { ContextMenu } from "components/ContextMenu"
-import { useMemo } from "react"
-import {
-  BiChevronRight,
-  BiEditAlt,
-  BiFolder,
-  BiTrash,
-  BiWrench,
-} from "react-icons/bi"
-import { Link as RouterLink, useRouteMatch } from "react-router-dom"
+import { Box } from "@chakra-ui/react"
 
 import elementStyles from "styles/isomer-cms/Elements.module.scss"
 import contentStyles from "styles/isomer-cms/pages/Content.module.scss"
 
 import { pageFileNameToTitle } from "utils"
 
-const FolderItem = ({ item, isDisabled }) => {
-  const { url } = useRouteMatch()
-
-  const { name, type, children } = item
-  const encodedName = encodeURIComponent(name)
-
-  const generatedLink = useMemo(() => {
-    return type === "dir"
-      ? `${url}/subfolders/${encodedName}`
-      : `${url}/editPage/${encodedName}`
-  }, [encodedName, type, url])
+const FolderItem = ({ item }) => {
```

Comment:
If we are removing `isDisabled` as an argument for this function, I believe we should also remove the corresponding parameter `isDisabled` [here](https://github.com/isomerpages/isomercms-frontend/blob/fix/reorering/src/components/folders/ReorderingModal.jsx#L115)

Reply:
good catch! ts would've caught this... sadly this file is in jsx so it's hard to tell what's omitted @__@